cs/src/Management/TunnelManagementClient.cs

```diff
@@ -18,6 +18,8 @@
 using System.Threading.Tasks;
 using System.Web;
 using Microsoft.DevTunnels.Contracts;
+using Microsoft.DevTunnels.Ssh;
```

Comment:
Note: these errors will be resolved when I update to use the SSH package. In my local testing I've included a link to  the project reference. Requires the companion SSH PR to be checked in first.